gum/backend-qnx/gumprocess-qnx.c

```diff
@@ -108,9 +111,94 @@ gum_process_modify_thread (GumThreadId thread_id,
                            GumModifyThreadFunc func,
                            gpointer user_data)
 {
-  g_assert_not_reached ();
+  gboolean success = FALSE;
+  struct sigaction action, old_action;
+
+  if ( thread_id == gum_process_get_current_thread_id ())
+  {
+    //TODO: don't know how to implement this.
+    //g_assert_not_reached ();
```

Comment:
All comments should be along the lines of `/* TODO: don't know how to implement this */`